python/src/intracktive/_tests/test_widget.py

```diff
@@ -0,0 +1,117 @@
+import webbrowser
+from typing import Callable
+from unittest.mock import patch
+
+import napari
+import pytest
+from intracktive._tests.test_convert import make_sample_data
```

Comment:
Hey @TeunHuijben , the `_tests` are often not packaged into the installation, so this might be causing the problem we see in the CI.

It would be more appropriate for it to be a fixture inside `conftest.py`, For example, [in napari](https://github.com/napari/napari/blob/main/napari/conftest.py)